docs/developers/tutorial.md

```diff
@@ -86,7 +85,7 @@ output/src
 ```
 
 Checkout the commit
-[55f9196163ab0a](https://github.com/olafurpg/named-literal-arguments/commit/55f9196163ab0a5dde22364ab1f7880bf4a5dc54),
+[55f9196163ab0a](https://github.com/scalacenter/scalafix-named-literal-arguments/commit/55f9196163ab0a5dde22364ab1f7880bf4a5dc54),
```

Comment:
I am wondering if we shouldn't rewrite the history to match the new APIs (instead of fixing it in a last commit together with the bump), WDYT?

Reply:
I have never done that.. why not! It's supposed to compile even when using old hash ! If we rewrite history, we need to rewrite more than one commit.. I don't know how many, but I guess it's doable. Do you think it's worth it ?

Reply:
Now that you have rewritten history, shouldn't this be https://github.com/scalacenter/scalafix-named-literal-arguments/commit/12dce5106332f8af5debaf2a0522a7f78c77fade?

All branches containing https://github.com/scalacenter/scalafix-named-literal-arguments/commit/55f9196163ab0a5dde22364ab1f7880bf4a5dc54 should probably be updated (particularly the step-* ?)

Reply:
I have updated the steps on the project